src/concerto/metamodel@0.3.0.cto

```diff
@@ -14,6 +14,20 @@
 
 namespace concerto.metamodel
 
+/**
+ * File location
+ */
+concept LocationPoint {
```

Comment:
Rename this to be `Location` or `Position`?

Reply:
I quite like the proposal, pushed and update to use `Range` and `Position`. All use is ` o Range location`